# Адрес ШИМ-выхода
int32 num

# Значение ШИМ
uint8 value